central/blob/datastore/store/store.go

```diff
@@ -51,6 +57,18 @@ func wrapRollback(ctx context.Context, tx *pgPkg.Tx, err error) error {
 	return err
 }
 
+func (s *storeImpl) Count(ctx context.Context, q *v1.Query) (int, error) {
+	defer metrics.SetIndexOperationDurationTime(time.Now(), ops.Count, "Blob")
```

Comment:
To which extent do we want to keep the `IndexOperationDurationTime` metrics. I think in the previous cases, they were replaced by `PostgresOperationDurationTime`.

Reply:
I think we should move to PostgresOp timing

Reply:
I don't know where these metrics are exposed.
If they are available to customers in any way, we should probably have dual metrics feeds for the duration of the deprecation notice, then remove all `IndexOp` collection from the code.

Reply:
This is an oversight. This should, in fact, be setting Postgres op duration.